pkg/helmcertifier/checks/checks.go

```diff
@@ -31,7 +31,7 @@ func IsHelmV3(uri string) (Result, error) {
 	if err != nil {
 		return Result{}, err
 	}
-	isHelmV3 := c.Metadata.APIVersion == "v3"
+	isHelmV3 := c.Metadata.APIVersion == "v2"
```

Comment:
In addition to the check itself, can you include also a message describing the result, and how to fix the issue if possible? You can also add a field named `Reason` in the the `Result` type; this will be handy later on when collecting the results and informing the user.

Reply:
I have added `Reason`.